RELEASE-NOTES.txt

```diff
@@ -9,6 +9,7 @@
 * [*] Block editor: Fixed a race condition when autosaving content [https://github.com/WordPress/gutenberg/pull/36072]
 * [**] Block editor: Image block: Add ability to quickly link images to Media Files and Attachment Pages [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3971]
 * [**] Block editor: Fixed a crash that could occur when copying lists from Microsoft Word. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4174]
+* [**] Fixed a crash that would occur if the user rotates the device when using the editor. [https://github.com/wordpress-mobile/WordPress-Android/pull/15579]
```

Comment:
👋 Joel! Looks like the issue under fix was introduced with 18.7 so, I think there's no need for an entry in the release-notes as by the time the beta gets promoted to production the issue will have been fixed, right?

Reply:
Indeed. I will remove the entry 🙇🏾‍♂️